LondonBritishLibrary/orient/BLorient656.xml

```diff
@@ -56,60 +90,67 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">229</measure>
-                                    
-                                    
-                                    
-                                    
-                                    
-                                    <dimensions type="outer" unit="mm">
-                                        <height>505</height>
-                                        <width>457</width>
+                                    <measure unit="blank">1</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>19.875</height>
+                                        <width>18</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+                                <foliation>One unnumbered blank leaf at the end.</foliation>
+
                             </supportDesc>
                             <layoutDesc>
                                 <layout columns="3" writtenLines="30 33" cert="high">
                                 </layout>
                             </layoutDesc>
-                            
-                            
+
+
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is written in a fine hand of the  <date>17th century</date>, 
-                                    
+                                <desc>The manuscript is written in a fine hand of the  <date>17th century</date>.
                                 </desc>
-                                
-                                <date notBefore="1600" notAfter="1700"/>
-                                
+
+                                <date notBefore="1600" notAfter="1667" evidence="internal"/>
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            
-                            
+<list>
+<item xml:id="e1">
+<locus target="#72r #200r"/>
+<desc>On <locus target="#72r">f. 72r</locus> and <locus target="#200r">f. 200r</locus> a name partially erased, perhaps that of the scribe or owner,
+  <foreign xml:lang="gez"><gap reason="illegible" resp="bm:Wright1877BM"></gap>... ጊዮርጊስ፡</foreign></desc>
```

Comment:
three dots are not needed, if you have encoded the gap. I also do not think that the resp is right here...